java/piranha/src/test/java/com/uber/piranha/XPFlagCleanerTest.java

```diff
@@ -396,6 +396,50 @@ public void positiveCaseWithFlagNameAsVariable() throws IOException {
     }
   }
 
+  @Test
+  public void positiveCaseWithMultipleControlArguments() throws IOException {
```

Comment:
Can you add a negative case also? One for flag name not being stale and another for not matching the method name.